presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -112,6 +112,20 @@ String Functions
 
     Converts ``string`` to uppercase.
 
+.. function:: lpad(string, size, filltext) -> varchar
```

Comment:
Let's rename `filltext` to `padtext` since the function is named "pad" rather than "fill".